src/containers/AskGlific/AskGlific.module.css

```
@@ -1,311 +1,726 @@
+.Panel {
+  box-sizing: border-box;
+}
+
+.Panel *,
+.Panel *::before,
+.Panel *::after {
+  box-sizing: border-box;
+}
+
+.FloatingWrapper {
+  composes: Panel;
```

Stylelint (17.14.0)

[error] Expected "Panel" to be "panel" (value-keyword-case)

(value-keyword-case)